docs/templates/schedule.md

```diff
@@ -0,0 +1,40 @@
+# Workspace Scheduling
+
+Templates can configure various aspects of how its workspaces are started and
+stopped, as well as manage the lifecycle of failed or inactive workspaces.
+
+![Schedule screen](../images/template-scheduling.png)
+
+## Schedule
+
+Template admins may define the default values for how long a workspace is
```

Comment:
How about this?

```
Template [admins](../admin/users.md) may define these default values:

- **Autostop**: How long a workspace runs without user activity before
  Coder automatically stops it.
- **Max lifetime**: The maximum duration a workspace stays in a
  started state before Coder forcibly stops it.
```

Rationale: Two distinct defaults, so let's split them into 2 distinct bullets. Let's refer to these options by their UI labels to make it easier for users to navigate the UI and search the help for related info. More explicitness about who does what.